__tests__/hooks/useGroupSetting.test.js

```diff
@@ -1,12 +1,11 @@
-import * as launchGallery from 'react-native-image-picker';
 import React from 'react';
-import SimpleToast from 'react-native-simple-toast';
+import * as launchGallery from 'react-native-image-picker';
 import {act, renderHook} from '@testing-library/react-hooks';
 
-import * as serviceFile from '../../src/service/file';
 import * as servicePermission from '../../src/utils/permission';
 import useGroupSetting from '../../src/screens/GroupSetting/hooks/useGroupSetting';
 import {Context} from '../../src/context/Store';
+import {Linking} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and `SimpleToast` from 'react-native-simple-toast' has been removed. If this module is no longer used in the code, this change is fine. However, if it's still being used somewhere else in the code, this could cause a ReferenceError.
```diff
- import SimpleToast from 'react-native-simple-toast';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->